kornia/geometry/transform/thin_plate_spline.py

```diff
@@ -79,19 +94,25 @@ def get_tps_transform(points_src: torch.Tensor, points_dst: torch.Tensor) -> tup
     pair_distance: torch.Tensor = _pair_square_euclidean(points_src, points_dst)
     k_matrix: torch.Tensor = _kernel_distance(pair_distance)
 
-    zero_mat: torch.Tensor = zeros(batch_size, 3, 3, device=device, dtype=dtype)
-    one_mat: torch.Tensor = ones(batch_size, num_points, 1, device=device, dtype=dtype)
+    zero_mat: torch.Tensor = torch.zeros(batch_size, 3, 3, device=device, dtype=dtype)
+    one_mat: torch.Tensor = torch.ones(batch_size, num_points, 1, device=device, dtype=dtype)
     dest_with_zeros: torch.Tensor = torch.cat((points_dst, zero_mat[:, :, :2]), 1)
     p_matrix: torch.Tensor = torch.cat((one_mat, points_src), -1)
     p_matrix_t: torch.Tensor = torch.cat((p_matrix, zero_mat), 1).transpose(1, 2)
-    l_matrix: torch.Tensor = torch.cat((k_matrix, p_matrix), -1)
+
+    if lambda_reg is not None and lambda_reg >= 0:
+        identity_matrix = torch.eye(num_points, device=device, dtype=dtype).unsqueeze(0).repeat(batch_size, 1, 1)
```

Comment:
```suggestion
        identity_matrix = eye_like(num_points)
```
`from kornia.utils import eye_like`